scripts/tokenomics-emulation/network.ts

```diff
@@ -236,7 +236,7 @@ class Network {
           throw e;
         }
         const seconds = parseInt(waitSeconds, 10);
-        console.log(`Wait ${seconds}s.`);
+        console.log(`Wait ${seconds} ms.`);
```

Comment:
Is it right name for variable `seconds`?

Reply:
updated.